docs/gradle-plugin/index.md

```diff
@@ -23,10 +27,12 @@
 
 * Collection of code coverage through `JVM` tests (JS and native targets are not supported yet).
 * generating `HTML` and `XML` reports.
+* Verification rules with bounds to keep track of coverage.
+* Create tasks to generate reports and automatically adjusts the dependencies between it
```

Comment:
```suggestion
* Create tasks to generate reports and automatically adjust the dependencies between it
```

To be consistent with the rest of the list. IMO, everything here should be in third-person, but that depends on the style